apps/sr-frontend/src/app/PageHome.tsx

```diff
@@ -10,6 +22,18 @@ export default function PageHome() {
     <Stack gap={1} pt={1}>
       <CharacterSelector />
       <CharacterEditor />
+      <Container>
+        <Accordion>
+          <AccordionSummary expandIcon={<ExpandMore />}>
+            Characters
+          </AccordionSummary>
+          <AccordionDetails>
+            <CharacterInventory />
+          </AccordionDetails>
+        </Accordion>
+      </Container>
+      <RelicEditor />
```

Comment:
The `RelicEditor` and `RelicInventory` has been moved out of this page. can you remove it in your PR as well?